publish2cloud.py

```diff
@@ -39,25 +41,14 @@
     REMOTE_SETTINGS_RECORD_PATH = ('/buckets/{bucket_name}'
                                    + '/collections/{collection_name}/records')
     REMOTE_SETTINGS_AUTH = ''
-    if rs_auth_method == "token":
-        if os.environ.get('SHAVAR_REMOTE_SETTINGS_TOKEN', None):
-            REMOTE_SETTINGS_AUTH = BearerAuth((
-                CONFIG.get('main', 'remote_settings_token')
-            ))
-    elif rs_auth_method == "userpass":
-        REMOTE_SETTINGS_AUTH = ('', '')
-        auth_config_exists = (
-            os.environ.get('SHAVAR_REMOTE_SETTINGS_USERNAME', None)
-            and os.environ.get('SHAVAR_REMOTE_SETTINGS_PASSWORD', None)
-        )
-        if auth_config_exists:
-            REMOTE_SETTINGS_AUTH = (
-                CONFIG.get('main', 'remote_settings_username'),
-                CONFIG.get('main', 'remote_settings_password')
-            )
-    else:
-        sys.stderr.write(
-            'No Auth method provided in environment variables\n')
+    if os.environ.get('AUTHORIZATION', None):
+        REMOTE_SETTINGS_AUTH = CONFIG.get('main', 'remote_settings_authorization')
+
+    # We can remove the use of BearerAuth() once we switch to kinto-http
+    if rs_auth_method == 'token':
+        REMOTE_SETTINGS_AUTH = BearerAuth(REMOTE_SETTINGS_AUTH)
+    elif rs_auth_method == 'userpass':
+        REMOTE_SETTINGS_AUTH = HTTPBasicAuth(tuple(REMOTE_SETTINGS_AUTH.split(":", maxsplit=1)))
```

Comment:
According the `requests` docs, you either have to do this:
`REMOTE_SETTINGS_AUTH = tuple(REMOTE_SETTINGS_AUTH.split(":", maxsplit=1))`
or this:
`REMOTE_SETTINGS_AUTH = HTTPBasicAuth(*tuple(REMOTE_SETTINGS_AUTH.split(":", maxsplit=1)))`

Because the `auth=` parameter down the line will support a simple tuple, but the `HTTPBasicAuth` constructor does not take a tuple as parameter, but two separate values, user, and password.

Reply:
Oh I see, made the change!